src/components/LandingPage/index.tsx

```diff
@@ -44,6 +45,10 @@ const LandingPage = ({ handleSelectDataset, megasets }: LandingPageProps) => (
                             ))}
                         </Row>
                     </div>
+
+                    <div className={styles.sectionContent}>
+                        <CsvInput />
+                    </div>
```

Comment:
what happens if you load a CSV and then select one of the existing datasets afterwards?

Reply:
Loading a CSV immediately takes you to the plot view, so you can no longer select the JSON or Firebase datasets.

Reply:
but if you hit back you can, and it's a SPA

Reply:
ack

Reply:
Opened a ticket as #221. I've started tinkering with this but need more time to make the solution not too janky.